src/workflow/units/execution/__init__.py

```diff
@@ -33,8 +34,8 @@ def parser_name(self):
         Returns:
              str: espresso or vasp
         """
-        if find_file(ExaParserConfig["global"]["vasp_xml_file"], self.work_dir): return "vasp"
-        if find_file(ExaParserConfig["global"]["espresso_xml_file"], self.work_dir): return "espresso"
+        if find_file(VASP_XML_FILE, self.work_dir): return "vasp"
```

Comment:
TODO: remove when a way to extract/construct workflows for vasp/espresso is implemented

Reply:
Done.